perfkitbenchmarker/vm_util.py

```diff
@@ -249,7 +249,8 @@ def WrappedFunction(*args, **kwargs):
             raise e
           else:
             if log_errors:
-              logging.error('Got exception running %s: %s', f.__name__, e)
+              logging.error('Got exception running %s: %s', f.__name__, e,
```

Comment:
How about this version - full stack is only printed after you've timeout/max-retried.  So if it succeeds, the logs are small. If it fails, you have what you need to debug (eventually).  And no configs to muck with.